openslides_backend/action/actions/motion/import_.py

```diff
@@ -0,0 +1,644 @@
+                            )
+                        elif (
+                            meeting_user_id in new_submitters.get(motion_id, {}).keys()
+                        ):
+                            sorted_motion_submitter_ids.append(
+                                new_submitters[motion_id][meeting_user_id]
+                            )
+                        else:
+                            raise Exception(
+                                f"Submitter sorting failed due to submitter for user/{submitter_user_id} not being found"
+                            )
+                    if len(sorted_motion_submitter_ids):
+                        sort_payload.append(
+                            {
+                                "motion_id": motion_id,
+                                "motion_submitter_ids": sorted_motion_submitter_ids,
+                            }
+                        )
+                for payload in sort_payload:
+                    self.execute_other_action(MotionSubmitterSort, [payload])
+
+        return {}
+
+    def get_ids_from_object_list(self, object_list: list[dict[str, Any]]) -> list[int]:
+        return [
+            obj["id"]
+            for obj in object_list
+            if obj.get("info") != ImportState.WARNING
+            and obj.get("info") != ImportState.ERROR
+        ]
+
+    def remove_fields_from_data(
+        self, data: dict[str, Any], fieldnames: list[str]
+    ) -> None:
+        for fieldname in fieldnames:
+            data.pop(fieldname, None)
+
+    def validate_entry(self, row: ImportRow) -> ImportRow:
+        entry = row["data"]
+
+        number = self.get_value_from_union_str_object(entry.get("number"))
+        if number:
+            check_result = self.number_lookup.check_duplicate(number)
+            id_ = cast(int, self.number_lookup.get_field_by_name(number, "id"))
+
+            if check_result == ResultType.FOUND_ID and id_ != 0:
+                if row["state"] != ImportState.DONE:
+                    row["messages"].append(
+                        f"Error: Row state expected to be '{ImportState.DONE}', but it is '{row['state']}'."
+                    )
+                    row["state"] = ImportState.ERROR
+                    entry["number"]["info"] = ImportState.ERROR
+                elif "id" not in entry:
+                    raise ActionException(
+                        f"Invalid JsonUpload data: A data row with state '{ImportState.DONE}' must have an 'id'"
+                    )
+                elif entry["id"] != id_:
+                    row["state"] = ImportState.ERROR
+                    entry["number"]["info"] = ImportState.ERROR
+                    row["messages"].append(
+                        f"Error: Number '{number}' found in different id ({id_} instead of {entry['id']})"
+                    )
+            elif check_result == ResultType.FOUND_MORE_IDS:
+                row["state"] = ImportState.ERROR
```

Comment:
write tests to provoke this error. There are different combinations possible, i.e 2 on database (technically possible), 2 in payload